webpack/components/TimelineIndicator.jsx

```diff
@@ -50,7 +52,10 @@ class TimelineIndicator extends Component {
           position={{ x: this.calculateProgress(), y: 0 }}
           ref={this.indicator}
           axis="x"
-          bounds="parent"
+          bounds={{
```

Comment:
These changes, along with the debouncing, fix the level 1 timeline drag issue on all of the browsers I've tested (FF, Chrome, Safari for Mac).